### ISSUE TITLE
关于论文Figure 5的疑问

### ISSUE THREAD

**Issue by qq31415926** [lancopku/label-words-are-anchors#22]
您好，请问论文中Figure 5的auc score具体是如何计算的？为啥auc score会表示注意力矩阵和预测的相关程度？

**Comment by leanwang326** [comment]
就是attention矩阵中target position对于label token位置的attention作为对应类的y_score（sklearn.metrics.roc_auc_score(y_true, y_score）里的y_score），真实标签作为y_true，去计算arc roc score，表示了target position对于label token位置的attention的大小和预测的相关程度

**Comment by qq31415926** [comment]
非常感谢，另外还想问一下在两个基座gpt-xl和gpt-j上进行reweighting实验分别需要多少显存？

**Comment by leanwang326** [comment]
gpt2-xl float32实测应该 27GB左右，gpt-j-6b float16的话应该28GB左右
